src/test/scala/org/broadinstitute/dsde/firecloud/service/SubmissionServiceNegativeSpec.scala

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import org.broadinstitute.dsde.firecloud.FireCloudConfig
+import org.broadinstitute.dsde.firecloud.mock.MockWorkspaceServer
+import spray.http.HttpMethods.{GET, DELETE, POST}
+
+/**
+  * We don't create a mock server so we can differentiate between methods that get passed through (and result in
+  * InternalServerError) and those that aren't passed through at the first place (i.e. not 'handled')
+  */
+final class SubmissionServiceNegativeSpec extends ServiceSpec with SubmissionService {
+
+  def actorRefFactory = system
+
+  val localSubmissionsPath = FireCloudConfig.Rawls.submissionsPath.format(
+    MockWorkspaceServer.mockValidWorkspace.namespace,
+    MockWorkspaceServer.mockValidWorkspace.name)
+
+  val localSubmissionIdPath = FireCloudConfig.Rawls.submissionsIdPath.format(
+    MockWorkspaceServer.mockValidWorkspace.namespace,
+    MockWorkspaceServer.mockValidWorkspace.name,
+    MockWorkspaceServer.mockValidId)
+
+  "SubmissionService" - {
+    "non-GET requests hitting the /submissions/queueStatus path are not passed through" in {
+      allHttpMethodsExcept(GET) foreach { method =>
+        checkIfPassedThrough(routes, method, "/submissions/queueStatus", toBeHandled = false)
+      }
+    }
+
+    "non-GET/POST requests hitting the /workspaces/*/*/submissions path are not passed through" in {
+      allHttpMethodsExcept(GET, POST) foreach { method =>
+        checkIfPassedThrough(routes, method, localSubmissionsPath, toBeHandled = false)
+      }
+    }
+
+//    "non-POST requests hitting the /workspaces/*/*/submissions/validate path are not passed through path are not passed through" in {
+//      allHttpMethodsExcept(POST) foreach { method =>
+//        checkIfPassedThrough(routes, method, s"$localSubmissionsPath/validate", toBeHandled = false)
+//      }
+//    }
```

Comment:
Do you have any thoughts on why the block commented out above may be failing?
@aednichols @davidangb

Reply:
It's strange that the test still fails even if I delete the whole block
```
path("validate") {
  post {
    passthrough(s"$workspacesUrl/$namespace/$name/submissions/validate", POST)
  }
} ~
```
from `SubmissionService.scala`

Reply:
I believe the path `/workspaces/namespace/name/submissions/validate` is being passed through on line 40 with a `submissionId` of `"validate"`

Reply:
You're right on. Thank you!